channels/utils.py

```diff
@@ -1,9 +1,12 @@
 import asyncio
 import types
 from concurrent.futures import CancelledError
+from typing import Any, Callable, Dict, List
 
+StrDict = Dict[str, Any]
```

Comment:
type defs like this might be better for us to not put in the `until` file but rather in a types file?

also even through the type of `scope` is a `StrDict` it might be cleaner for reading if we redoing a `Scope` type here?

Reply:
I agree with the scope type, and I think it should go in `typing.py` file

Reply:
I would suggest in that `module` have a few aliases eg things like:

```python
Scope = StrDict
```

Reply:
Okay, I can definitely do that

Reply:
If we're having `typing` then `Scope = Dict[str, Any]` is fine I think. (We can probably use a union type to narrow that down further, but... (later for that.)